mr/src/main/java/org/opensearch/hadoop/util/OpenSearchMajorVersion.java

```diff
@@ -45,8 +45,8 @@ public class OpenSearchMajorVersion implements Serializable {
     public static final OpenSearchMajorVersion V_5_X = new OpenSearchMajorVersion((byte) 5, "5.x");
```

Comment:
This version modelling is very confusing, shouldn't we split it off to ES / OpenSearch?